rpcs3/basic_keyboard_handler.h

```diff
@@ -18,6 +18,7 @@ class basic_keyboard_handler final : public QObject, public KeyboardHandlerBase
 	bool eventFilter(QObject* obj, QEvent* ev) override;
 	void keyPressEvent(QKeyEvent* event);
 	void keyReleaseEvent(QKeyEvent* event);
+	int get_converted_key(QKeyEvent* event);
```

Comment:
I'm a wee bit triggered by the mixed formatting in this module. Just a nit though.